README.md

```diff
@@ -225,7 +216,7 @@ For example:
     node build.js modules=ALL exclude=json no-strict no-svg-export
 
 
-### Goals
+## Goals
```

Comment:
I think this is more `Guidelines` than `Goals` because they are acheived.

Reply:
Achievements? 😄  Section contains great stuff, but I don't know how to name it properly.